spec/requests/item_presenter_spec.rb

```diff
@@ -23,4 +23,12 @@
     expect(presented_item).to include(:sellIn)
     expect(presented_item).not_to include(:sell_in)
   end
+  it 'converts camelCase attributes to snake case for saving to the database' do
```

Comment:
Request for changes: You should add some more test coverage here. What happens if...

* A field is nil?
* A field is missing entirely from the params?
* A field is an empty string?
* There are extra fields in the hash?

Also, I notice that you are using FactoryBot to create an item which is used to indirectly test the presenter. Why can't you just test the presenter directly? Pass in a hash of `{name: "whatever", sellIn: X, quality: Y}` and test the output.

FactoryBot is a powerful tool, to be sure, but the HUGE downside of it is it is super slow. If you use FactoryBot in a lot of tests, your test suite will literally grind to a halt

Reply:
Eric might have mentioned this to you during the pairing session, and I echo his sentiment. I don't like using FactoryBot because I think it encourages you to create database objects for tests when they are not needed. I am not saying you should not use it in this project (it is great in certain situations), but really think about if you need to touch the database in a test before using it.